packages/app/pages/index.tsx

```diff
@@ -17,14 +17,25 @@ import { AbiType, StateMutabilityType, AbiItem } from "web3-utils";
 import { Modal, Input, Button } from "@cabindao/topo";
 import { styled } from "../stitches.config";
 import BN from "bn.js";
-import passportFactoryJson from "../../contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
-import passportJson from "../../contracts/artifacts/contracts/Passport.sol/Passport.json";
+import passportFactoryJson from "@cabindao/nft-passport-contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
+import passportJson from "@cabindao/nft-passport-contracts/artifacts/contracts/Passport.sol/Passport.json";
```

Comment:
Could organize the artifacts in a future iteration so that the json files are in the root